turbo/consumers.py

```diff
@@ -2,18 +2,15 @@
 from asgiref.sync import async_to_sync
```

Comment:
In this file I'm removing the `request_id` mechanism. I wasn't sure what it was doing, but I wanted to start a discussion about it. Instead I'm using the `channel_name` (not to be confused with `self.channel_name` which is a Channels construct) as a group to send updates about models to specific clients. This more closely follows what the example chat app in the Channels docs does, but allows muliplexing multiple subscriptions over the same consumer. @davish, would you be able to shed some light on the `request_id` architecture?

Reply:
`request_id` was more of a holdover from another websockets project I'm working on where there's a few more layers of indirection -- I included it here originally so that the template tag could specify the `dom_id` and template partial for a given stream per-request, but we're handling that in the mixin/fully on the backend now. So it's not really necessary, and sorry it's causing bugs :/

Reply:
No worries! Thank you for the explanation. Sounds like we can remove it for now?

Reply:
Yep! can be removed